libqtile/extension/command_set.py

```diff
@@ -62,24 +62,29 @@ class CommandSet(Dmenu):
     """
 
     defaults = [
-        ("commands", None, "dictionary of commands where key is runable command"),
+        ("commandset", None, "dictionary of commands where key is runable command"),
```

Comment:
commanddict? but can we just leave this as commands, or is there a technical reason not to?

Reply:
Yes - it's an annoying clash.

And, yes, it's a dict. Will fix.